projects/frontend/data-pipelines/gui/package.json

```diff
@@ -0,0 +1,141 @@
+{
+  "name": "data-pipelines-wrapper",
+  "version": "0.0.1",
+  "engines": {
+    "node": "^14.15.0 || >=16.10.0",
+    "npm": ">=8.5.5"
+  },
+  "private": true,
+  "scripts": {
+    "ng": "node --max_old_space_size=2048 ./node_modules/@angular/cli/bin/ng",
+    "start": "npm run ng -- serve",
+    "build": "npm run ng -- build",
+    "build:dev": "npm run build -- --configuration=development",
+    "build:watch": "npm run ng -- build --watch",
+    "build:ui": "npm run build -- ui --configuration=production",
+    "test": "npm run ng -- test --watch=false",
+    "test:headless": "npm run test -- --browsers=ChromeHeadless_No_Sandbox",
+    "lint": "npm run ng -- lint",
+    "e2e": "npx cypress run --browser chrome --env OAUTH2_API_TOKEN=${VDK_API_TOKEN},CYPRESS_TERMINAL_LOGS='./e2e/logs'",
```

Comment:
Is this api token env var documented somewhere? Like where to get it and how to set and use it, default values etc.

Reply:
It's an existing one, in gitlab ci/cd variables,
so this will be documented and used in the scope of the CI/CD change (upcoming)